docs/setup/deploymentnew/v3-installation/1.2.1.0/on-premises-deployment-without-dns.md

```
@@ -0,0 +1,1353 @@
+
+    > Note:
+    >
+    > * Script prompts for below mentioned inputs please provide as and when needed:
+    >   * Enter the time (hr) to run the cronjob every day (0–23): Specify the hour you want the cronjob to run (e.g., 6 for 6 AM)
+    >   * Do you have a public domain and valid SSL certificate? (Y/n):
+    >     * Y – If you have a public domain and valid SSL certificate
+    >     * n – If you do not have one (recommended only for development environments)
+    >   * Retention days to remove old reports (Default: 3): Press Enter to accept the default or specify another value (e.g., 5).
+    >   *   Provide Slack Webhook URL to notify server issues on your Slack channel: (change the URL to your channel one)
+    >
+    >       ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)